web/tests/stream_list.test.cjs

```diff
@@ -337,6 +360,7 @@ const carSub = {
     pin_to_top: false,
     subscribed: true,
     is_recently_active: false,
+    can_send_message_group: everyone_group.id,
 };
```

Comment:
We should be trying to resolve things like this by using `web/tests/lib/example_stream.cjs` and extending that ... but maybe it'll require some further preparatory work to create `everyone_group` first, so nothing to do for right now.